assets/sass/backoffice/_icons.scss

```diff
@@ -0,0 +1,159 @@
+}
+
+%icon-conversations {
+    background: url("data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMjQiIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgZmlsbD0ibm9uZSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KPHBhdGggZmlsbC1ydWxlPSJldmVub2RkIiBjbGlwLXJ1bGU9ImV2ZW5vZGQiIGQ9Ik0xMSAyQzEwLjQ2OTYgMiA5Ljk2MDg2IDIuMjEwNzEgOS41ODU3OSAyLjU4NTc5QzkuMjEwNzEgMi45NjA4NiA5IDMuNDY5NTcgOSA0VjEwQzkgMTAuNTMwNCA5LjIxMDcxIDExLjAzOTEgOS41ODU3OSAxMS40MTQyQzkuOTYwODYgMTEuNzg5MyAxMC40Njk2IDEyIDExIDEySDE3LjU4NThMMjAuMjkyOSAxNC43MDcxQzIwLjU3ODkgMTQuOTkzMSAyMS4wMDkgMTUuMDc4NyAyMS4zODI3IDE0LjkyMzlDMjEuNzU2NCAxNC43NjkxIDIyIDE0LjQwNDUgMjIgMTRWNEMyMiAzLjQ2OTU3IDIxLjc4OTMgMi45NjA4NiAyMS40MTQyIDIuNTg1NzlDMjEuMDM5MSAyLjIxMDcxIDIwLjUzMDQgMiAyMCAySDExWk0xMSA0SDIwVjExLjU4NThMMTguNzA3MSAxMC4yOTI5QzE4LjUxOTYgMTAuMTA1NCAxOC4yNjUyIDEwIDE4IDEwSDExVjRaTTQgOUMzLjQ2OTU3IDkgMi45NjA4NiA5LjIxMDcxIDIuNTg1NzkgOS41ODU3OUMyLjIxMDcxIDkuOTYwODYgMiAxMC40Njk2IDIgMTFWMjFDMiAyMS40MDQ1IDIuMjQzNjQgMjEuNzY5MSAyLjYxNzMyIDIxLjkyMzlDMi45OTA5OSAyMi4wNzg3IDMuNDIxMTEgMjEuOTkzMSAzLjcwNzExIDIxLjcwNzFMNi40MTQyMSAxOUgxM0MxMy41MzA0IDE5IDE0LjAzOTEgMTguNzg5MyAxNC40MTQyIDE4LjQxNDJDMTQuNzg5MyAxOC4wMzkxIDE1IDE3LjUzMDQgMTUgMTdWMTVDMTUgMTQuNDQ3NyAxNC41NTIzIDE0IDE0IDE0QzEzLjQ0NzcgMTQgMTMgMTQuNDQ3NyAxMyAxNVYxN0g2QzUuNzM0NzggMTcgNS40ODA0MyAxNy4xMDU0IDUuMjkyODkgMTcuMjkyOUw0IDE4LjU4NThWMTFINkM2LjU1MjI4IDExIDcgMTAuNTUyMyA3IDEwQzcgOS40NDc3MiA2LjU1MjI4IDkgNiA5SDRaIiBmaWxsPSIjMjUyRDQwIi8+Cjwvc3ZnPgo=") no-repeat center center;
+}
+
+%icon-event-admin {
```

Comment:
Should be removed.

Reply:
done here https://github.com/netgen/media-site/commit/f5531f7d7dea7ccb70e555fa113a0e299641e8fb